### PR TITLE
change status to uppercase

### PR DESCRIPTION
**Pull request by LukeANewton**
this is a fix for issue #104

### CODE REVIEW

**Comment by CameronRushton** [comment]
I'm getting a new error on this branch. It's a stack overflow exception to do with User and Application since theres a circular dependency. 
[SOE.txt](https://github.com/CameronRushton/Graduate-Admission-System/files/4390055/SOE.txt)

**Comment by LukeANewton** [comment]
Is you thing all up to date? I am not getting these the error this issue is for, or the new issue when I run locally

**Comment by CameronRushton** [comment]
Yeah, I'm up to date. The staging app also doesn't start up.
image

**Comment by LukeANewton** [comment]
@CameronRushton does this fix the other error? I removed the applicant field in application since we dont really need that

**Comment by CameronRushton** [comment]
Yes, this fixes the other error #106 .

File: src/test/java/sysc4806/graduateAdmissions/controller/ApplicationControllerTest.java [hunk 1]
```diff
@@ -73,16 +77,18 @@ void setUpMocks() {
...
-        when(repo.findByApplicant_id(applicant2.getId())).thenReturn(Collections.singletonList(applications.get(2)));
+                new Application(0, term, Department.ELEC, degree,  professors, status, gpa, resumeFileName),
+                new Application(1, term, Department.SYSC, degree,  professors, status, gpa, resumeFileName),
+                new Application(2, term, Department.SREE, degree,  professors, status, gpa, resumeFileName));
+        applicant1.setApplications(new HashSet<>(applications.subList(0, 2)));
+        applicant2.setApplications(new HashSet<>(Collections.singletonList(applications.get(2))));
+
+        when(applicationRepository.findAll()).thenReturn(applications);
+        when(userRepository.findById(applicant1.getId())).thenReturn(Optional.ofNullable(applicant1));
+        when(userRepository.findById(applicant2.getId())).thenReturn(Optional.ofNullable(applicant2));
```
Comment (CameronRushton) [inline src/test/java/sysc4806/graduateAdmissions/controller/ApplicationControllerTest.java]: This invalidates most of the testing done here, but we're short on time and I'd like to start working again.

No changes required for this PR

**Pull request merged**